src/base-component/BaseComponent.jsx

```diff
@@ -21,7 +22,7 @@ const BaseComponent = ({ children, showWelcomeBanner }) => {
 
   return (
     <>
-      <CookiePolicyBanner languageCode={getLocale()} />
+      {getConfig().ENABLE_COOKIE_POLICY_BANNER ? <CookiePolicyBanner languageCode={getLocale()} /> : null}
```

Comment:
Please add a default (empty string) value for `ENABLE_COOKIE_POLICY_BANNER ` in `.env` file as this would result in undefined when no config is found